libs/prelude/Prelude/Strings.idr

```diff
@@ -132,6 +132,25 @@ pack = foldr strCons ""
 singleton : Char -> String
 singleton c = strCons c ""
 
+||| Adds a character to the end of the specified string.
+|||
+||| ```idris example
+||| strAppend "AB" 'C'
+||| ```
+||| ```idris example
+||| strAppend "" 'A'
+||| ```
+strAppend : String -> Char -> String
+strAppend s c = s ++ (singleton 'c')
+
+||| Alias of `strAppend`
+|||
+||| ```idris example
+||| "AB" +> 'C'
+||| ```
+(+) : String -> Char -> String
```

Comment:
You will probably have to declare the [fixity of the operator](http://idris.readthedocs.org/en/latest/tutorial/typesfuns.html?highlight=infix#data-types).